day_two_guide/topics/proc_backing-up-master.adoc

```diff
@@ -30,9 +30,6 @@ You can customize {product-title} services, such as increasing the log level or
 using proxies. The configuration files are stored in the `/etc/sysconfig`
 directory.
 
-Because the masters are also unschedulable nodes, back up the entire
-`/etc/origin` directory.
```

Comment:
I think this could just be a change of s/unschedulable/schedulable/, leaving the recommendation to back up all of `/etc/origin/` in.

Reply:
Or better yet, s/unschedulable//.

But if the line is taken out, it may also mean making step 1 in the procedure itself more specific.

Reply:
Thanks @adellape!